src/styles/price.scss

```diff
@@ -0,0 +1,18 @@
+.card__price {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+  line-height: 18px;
+  margin-top: 22px;
+
+  &-title {
+    font-size: 12px;
+    color: $description-color;
+  }
+
+  &-value {
+    font-size: 16px;
+    font-weight: bold;
+    color: $nav-links-color;
+  }
+}
```

Comment:
Why reviews are in the cards.scss file and the price is in separate, better to be consistent